Api/Controllers/CommentController.cs

```diff
@@ -53,4 +50,24 @@ public ActionResult DeleteComment([FromRoute] Guid id)
         commentService.AsUser(User.GetGuid()).DeleteComment(id);
         return Ok();
     }
+
+    [HttpGet]
+    [Authorize]
+    [Route("{id}/likes")]
+    [SwaggerOperation("List all comments's likes and dislikes")]
+    public ActionResult<IEnumerable<LikeDto>> GetCommentLikes([FromRoute] Guid id)
+    {
+        var reactions = commentService.AsUser(User.GetGuid()).GetComment(id).Likes;
+        return Ok(reactions.Select(x => x.ToDto()));
+    }
```

Comment:
I am not sure about returning all the likes if we don't know how many of them are there. Also we could send information about how many likes and dislikes are there (so frontend doesn't have to enumerate the whole list).